### PR TITLE
Zooming! English Readable Select Values, Gooninator URL Import, De-Dupe List

### PR DESCRIPTION
**Pull request by StarrHelixx**
Some initial work 👍

* Created a simple map for i18n mapping (this could def be improved)
* Added zoom capability (same as original gooninator)
* Made small change to CSS (make select spacing consistent and limit option bar width)
* Made change to image filter to _exclude_ `.gif` when "stills" is selected
* Added IntelliJ files to `.gitignore`
* Added 60fps option to timing values
* Added "- Remove All" option to Scene
  * **Plan to improve with confirmation dialog**
* Added "+ Import From URL" option to Scene
  * This takes a old gooninator url (really hostname is irrelevant, it just grabs the `sources` param)
  * Takes a parent dir where these blogs are located
  * Adds all these blogs to the list (de-dupes, but doesn't verify existence)
  * **Plan to improve with actual file selector (to pick parent dir instead of typing path)**

CSS for Modal could probably be improved too

### CODE REVIEW

**Comment by ififfy** [comment]
HOLY SHIT

**Comment by ififfy** [comment]
One more thing - I suggest making a branch before doing work, otherwise you might end up in a weird state later, if something unexpected happens to the master branch

File: src/renderer/components/DirectoryPicker.tsx [hunk 1]
```diff
@@ -29,19 +34,97 @@ export default class DirectoryPicker extends React.Component {
...
+          <input type="text" name="root" value={this.state.rootDir} onChange={this.rootDirChange}/>
+          <button onClick={this.doImport}>
+            Import
+          </button>
+        </Modal>
       </div>
     )
   }
 
+  // @ts-ignore
```
[the remark this answers is not in the document]
Comment (GoddessServant0) [reply src/renderer/components/DirectoryPicker.tsx]: I'm new to TypeScript, so trying to properly type where I can, I wasn't sure what type this was, so thank you very much! 👍

File: src/renderer/components/DirectoryPicker.tsx [hunk 2]
```diff
@@ -29,19 +34,97 @@ export default class DirectoryPicker extends React.Component {
...
+      isOpen: !this.state.isOpen,
+      importURL: ""
+    });
+    if (!importURL) {
+      return;
+    }
+    let rootDir = this.state.rootDir;
+    if (!rootDir.endsWith("\\")) {
+      rootDir += "\\";
+    }
```
[the remark this answers is not in the document]
Comment (GoddessServant0) [reply src/renderer/components/DirectoryPicker.tsx]: Yea, I was planning on fixing this before commit, totally spaced 😣

File: src/renderer/components/DirectoryPicker.tsx [hunk 3]
```diff
@@ -29,19 +34,97 @@ export default class DirectoryPicker extends React.Component {
...
   onAdd() {
     const result = remote.dialog.showOpenDialog({properties: ['openDirectory', 'multiSelections']});
     if (result) {
+      for (let u=0; u < result.length; u++) {
+        if (this.props.directories.includes(result[u])) {
+          // Remove index and push u back
+          result.splice(u,1);
+          u-=1
+        }
+      }
```
[the remark this answers is not in the document]
Comment (GoddessServant0) [reply src/renderer/components/DirectoryPicker.tsx]: Yes, ty, derp

**Pull request merged**